concent_api/core/views.py

```diff
@@ -653,6 +659,78 @@ def handle_send_force_subtask_results_results_response(request, client_message):
+    client_public_key       = decode_client_public_key(request)
+    other_party_public_key  = decode_other_party_public_key(request)
+    current_time            = get_current_utc_timestamp()
+
+    if not verify_message_subtask_results_accepted(client_message.subtask_results_accepted_list):
+        return message.concents.ServiceRefused(
+            reason = message.concents.ServiceRefused.REASON.InvalidRequest
+        )
+    requestor_ethereum_public_key = client_message.subtask_results_accepted_list[0].task_to_compute.requestor_ethereum_public_key
+
+    # Concent defines time T0 equal to oldest payment_ts from passed SubtaskResultAccepted messages from subtask_results_accepted_list.
+    T0 = min(subtask_results_accepted.payment_ts for subtask_results_accepted in client_message.subtask_results_accepted_list)
+
+    # Concent gets list of transactions from payment API where timestamp >= T0.
+    list_of_transactions = base.get_list_of_transactions(current_time = current_time, request = request)  # pylint: disable=no-value-for-parameter
+
+    # Concent defines time T1 equal to youngest timestamp from list of transactions.
+    T1 = max(transaction['timestamp'] for transaction in list_of_transactions)
+
+    # Concent checks if all passed SubtaskResultAccepted messages from subtask_results_accepted_list have payment_ts < T1
+    T1_is_bigger_than_payments_ts = any(T1 > subtask_results_accepted.payment_ts for subtask_results_accepted in client_message.subtask_results_accepted_list)
+
+    # Any of the items from list of overdue acceptances matches condition current_time < payment_ts + PAYMENT_DUE_TIME + PAYMENT_GRACE_PERIOD.
+    acceptance_time_overdue = any(current_time < subtask_results_accepted.payment_ts + settings.PAYMENT_DUE_TIME + settings.PAYMENT_GRACE_PERIOD for subtask_results_accepted in client_message.subtask_results_accepted_list)
+
+    if T1_is_bigger_than_payments_ts or acceptance_time_overdue:
+        return message.concents.ForcePaymentRejected(
+            reason = message.concents.ForcePaymentRejected.REASON.TimestampError
+        )
+
+    # Concent gets list of list of forced payments from payment API where T0 <= payment_ts + PAYMENT_DUE_TIME + PAYMENT_GRACE_PERIOD.
+    list_of_forced_payments = base.get_forced_payments(T0, requestor_ethereum_public_key, client_public_key, request = request)
+
+    V = base.payment_summary(request = request, subtask_results_accepted_list = client_message.subtask_results_accepted_list, list_of_transactions = list_of_transactions, list_of_forced_payments = list_of_forced_payments)  # pylint: disable=no-value-for-parameter
+
+    # Concent defines time T2 (end time) equal to youngest payment_ts from passed SubtaskResultAccepted messages from subtask_results_accepted_list.
+    min(subtask_results_accepted.payment_ts for subtask_results_accepted in client_message.subtask_results_accepted_list)
```

Comment:
This value should be used as payment_ts in `make_payment_to_provider` and ForcePaymentCommitted message. It is not described in blueprints so if we couldn't make it now I will create separate issue for this.